packages/notifier-seeder/src/models/catalogItemEventNotificationMessage.ts

```diff
@@ -0,0 +1,71 @@
+import { EServiceEventEnvelopeV2 } from "pagopa-interop-models";
+import { match } from "ts-pattern";
+import { v4 as uuidv4 } from "uuid";
+import { QueueMessage } from "../queue-manager/queueMessage.js";
+import { CatalogItemEventNotification } from "./catalogItemEventNotification.js";
+
+export const eventV2TypeMapper = (
+  eventType: EServiceEventEnvelopeV2["type"]
+): string =>
+  match(eventType)
+    .with("EServiceAdded", () => "catalog_item_added")
+    .with("DraftEServiceUpdated", () => "catalog_item_updated")
+    .with("EServiceDeleted", () => "catalog_item_deleted")
+    .with("EServiceCloned", () => "cloned_catalog_item_added")
+    .with("EServiceDescriptorAdded", () => "catalog_item_descriptor_added")
+    .with(
+      "EServiceDraftDescriptorUpdated",
```

Comment:
`EServiceDraftDescriptorUpdated` should produce a `catalog_item_updated`.
`catalog_item_descriptor_updated` is used for state change on already published descriptors. These events require the m2m notification.
While `catalog_item_updated` is used for draft elements changes, and do not produce a notification